src/content/release-notes.mdx

```diff
@@ -5,6 +5,40 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.24
+
+9 September 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Upgraded ingress-nginx to v4.11.2  <!-- 8460 -->
+
+### New Features
+
+- Add provider credentials query for deploy and test executions  <!-- https://github.com/okteto/app/pull/8454 -->
+- Enable [multi-user token authentication](self-hosted/install/auth/token.mdx)  <!-- https://github.com/okteto/app/pull/8481 -->
+- Announcing Okteto CLI 3.0 [Will link to blog post]
+
+### Improvements
```

Comment:
We always mention the upgrade to the Okteto CLI version. In this case, we are upgrading to CLI 2.31.0.
The main features are:
- Replace RSA by ECDSA for SSH keys. This allows the use of debuggers with Visual Studio with Development Containers
- Prevent syncthing to restart and activating [relay](https://docs.syncthing.net/users/relaying.html) if the synching configuration is lost